exercises/practice/ordinal-number/.meta/config.json

```
@@ -0,0 +1,12 @@
+{
+  "authors": [],
```

add your exercism name here:
```suggestion
  "authors": [
    {
      "github_username": "codedge",
      "exercism_username": "______"
    }
  ],
  "contributors": [
    {
      "github_username": "neenjaw",
      "exercism_username": "neenjaw"
    }
  ],
```

---

Fixed.